master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml

```diff
@@ -14,6 +14,10 @@ metadata:
   name: calico-config
   namespace: kube-system
 data:
+  # To enable Typha, set this to "calico-typha" *and* set a non-zero value for Typha replicas
+  # below.  We recommend using Typha if you have more than 50 nodes. Above 100 nodes it is
+  # essential.
+  typha_service_name: "none"
```

Comment:
huh, didn't realize Felix had a special case for "none" - is that the default? Or should it actually be `""`?

Reply:
It does have a special case for "none".  In this particular case, "none" and "" would do the same thing, I think.